libretroshare/src/chat/p3chatservice.cc

```diff
@@ -855,12 +855,8 @@ bool p3ChatService::handleRecvChatMsgItem(RsChatMsgItem *& ci)
         {
 #ifdef RS_DIRECT_CHAT
 			/* notify public chat message */
-			RsServer::notify()->AddPopupMessage(
-			            RS_POPUP_GROUPCHAT,
-			            ci->PeerId().toStdString(), "", message );
-			RsServer::notify()->AddFeedItem(
-			            RS_FEED_ITEM_CHAT_NEW,
-			            ci->PeerId().toStdString(), message, "" );
+			RsServer::notify()->AddPopupMessage( RS_POPUP_GROUPCHAT, ci->PeerId().toStdString(), "", message );
```

Comment:
is this still necessary?

Reply:
I removed all addFeedItem(), but I did not take care of the popup messages in this PR